front50-gcs/src/test/kotlin/com/netflix/spinnaker/front50/model/FakeStorageRpc.kt

```diff
@@ -57,11 +57,12 @@ internal class FakeStorageRpc(private val clock: Clock) : StorageRpc {
 
   private data class Blob(val storageObject: StorageObject, val content: ByteArray)
 
-  private val blobs = mutableMapOf<String, MutableList<Blob>>()
+  private val buckets = mutableMapOf<String, MutableMap<String, MutableList<Blob>>>()
```

Comment:
> We'll add an additional Map around the blobs data structure that maps the bucket to a Map of blobs. 

This is maybe a just-me-problem because I've never looked at any of this code until this morning, but if we aren't going to type this with a class can we either include this description from your commit message as a comment here, or rename `buckets` --> `bucketsToBlobs` or something that makes it a bit clearer what's in this nested data structure?

Reply:
Good call, looking at this again today it's less clear than it was to yesterday Eric.  Let me take a look at whether it's clearer to add a comment or just make a tiny inner class for this.

Reply:
Ok, I make a couple of small classes---let me know if this is clearer!

Reply:
Oh wow this is 💯 ! (But also, sorry, didn't mean for that comment to lead to this much more work for you!)

Reply:
No worries at all, this is way better and I learned some more kotlin along the way!